<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(233, 233, 233);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="closetaskButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>550</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: #40916c;
   border : 2px solid #40916c;
   border-radius :10px;
}
QPushButton:hover{
	color: #40916c;
   border : 2px solid #40916c;
   border-radius :10px;
   background-color : #b7e4c7;
}
QPushButton:pressed{
   	color: #40916c;
   border : 2px solid #40916c;
   border-radius :10px;
   background-color : #2d6a4f;
}

</string>
     </property>
     <property name="text">
      <string>Close Task</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>550</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
   color: #b21e35;
   border : 2px solid #b21e35;
   border-radius :10px;
}
QPushButton:hover{
   color: #b21e35;
   border : 2px solid #b21e35;
   border-radius :10px;
   background-color : #ff4d6d;
}
QPushButton:pressed{
   color: #b21e35;
   border : 2px solid #b21e35;
   border-radius :10px;
   background-color :#800f2f ;
}
</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>781</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time Start</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time Respon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>machine</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Problem</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comment</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>problem after check</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>solve</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>time finish</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>namemtc</string>
      </property>
     </column>
    </widget>
    <widget class="QTextEdit" name="probactext">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>771</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Problem After Check:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Solve :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="solvetext">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>771</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CLOSE TASK</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>988</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
